ipv8/test/attestation/wallet/test_attestation_community.py

```diff
@@ -200,6 +233,39 @@ def callback(rhash, values):
         self.assertTrue(callback.called)
         self.nodes[1].overlay.request_cache.clear()
 
+    @inlineCallbacks
+    def test_verify_attestation_range(self):
+        """
+        Check if an attestation can be verified for id_metadata_range_18plus.
+        """
+        filename = os.path.join(os.path.dirname(__file__), 'attestation_range.txt')
+        with open(filename, 'r') as f:
+            serialized = unhexlify(f.read().strip())
+        attestation = PengBaoAttestation.unserialize_private(self.private_key, serialized, "id_metadata_range_18plus")
+        attestation_hash = unhexlify('b40c8734ba6c91a49670c1f0152c7f4dac2a8272')
```

Comment:
would be easier to follow if the attestation hash is derived instead of hardcoded one.
`hashlib.sha1(attestation.serialize()).digest()` or `attestation.get_hash()`

Reply:
I agree, but considering the amount of logic, I follow this way of working in the attestation unit tests. The unit tests will simply take too long if we go full circle in every test. I compromised by using premade attestations.

Reply:
alright then